opensuse-tumbleweed-ports-repoindex.xml

```diff
@@ -22,6 +22,18 @@
     enabled="false"
     autorefresh="true"/>
 
+<repo url="%{disturl}/ports/$DIST_ARCH/%{distsub}/repo/non-oss"
+    alias="repo-non-oss"
+    name="%{alias}"
+    enabled="false"
+    autorefresh="true"/>
+
+<repo url="%{disturl}/ports/$DIST_ARCH/source/%{distsub}/repo/non-oss"
+    alias="repo-non-oss"
```

Comment:
Alias should be `repo-non-oss-source` but https://download.opensuse.org/ports/aarch64/source/tumbleweed/repo/non-oss/ does not exists.
(But it exists for `ppc` and `zsystems`.)
Also, not present on `x86_64` XML file.

Reply:
Ah yes nice catch!

Reply:
https://bugzilla.opensuse.org/show_bug.cgi?id=1226763